### PR TITLE
Remove cassandra-all

### PR DESCRIPTION
**Pull request by mjok**
this prone to collect vulnerable libs and sparsely used.
I think code copy would benefit more.

### CODE REVIEW

**Comment by maximevw** [comment]
Hello @mjok,
Thanks a lot for your pull request. I will review it as soon as possible.
Could you please modify it to merge into `release/next` branch as requested in our contributing policy? We´ll take advantage of your changes to build and release a new minor version (4.6.0) including other changes already included into this branch.

**Comment by maximevw** [comment]
Hello @mjok
Did you see the other comments on the pull request or didn't have you just the time to review them?

**Comment by mjok** [comment]
Didn't saw any comment other than branch to merge. Can't see any them either now

**Review by maximevw — CHANGES_REQUESTED** [review]

**Comment by maximevw** [comment]
> Didn't saw any comment other than branch to merge. Can't see any of them now. I did change the branch.

Sorry @mjok, my bad... I forgot to submit my review 😄

File: pom.xml [hunk 1]
```diff
@@ -96,26 +96,13 @@
...
-                    <artifactId>libthrift</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-        <!-- Force the use of a more recent version of libthrift. -->
-        <dependency>
-            <groupId>org.apache.thrift</groupId>
-            <artifactId>libthrift</artifactId>
-            <version>${libthrift.version}</version>
+            <groupId>com.google.guava</groupId>
```
[the remark this answers is not in the document]
Comment (mjok) [reply pom.xml:111]: com/ing/data/cassandra/jdbc/CassandraConnection.java:22
uses com.google.common.collect.Maps;

[the remark this answers is not in the document]
Comment (maximevw) [reply pom.xml:111]: Right. I missed it. 
Could you please manage the Guava version in the section `<properties>` of the POM file as for the other dependencies?
I note to patch this as soon as possible :)

**Review by maximevw — APPROVED** [review]
I'll prepare the release 4.6.0 very soon including these changes.
Thank you @mjok for your contribution.

**Pull request merged**